### ISSUE TITLE
Security concerns

### ISSUE THREAD

**Issue by Comeza** [TimoA200/airpod_counter#14]
I couldn't find any information on how the app handles personal information

**Comment by webtheo** [comment]
Yeah, that is not GDPR-compliant! Additionally, I miss an imprint so I know who operates this app.
